doc/source/installation.rst

```diff
@@ -53,6 +53,29 @@ See the documentation of `pip <https://pip.pypa.io/en/stable/>`_ for more inform
 You may also want to look at `virtualenv <https://virtualenv.readthedocs.org/en/latest/>`_.
 
 
+Testing PBxplore
+----------------
+
+`PBxplore` comes with unit tests and regression tests. It requires the package
+`nose <https://nose.readthedocs.io/en/latest/>`_ to run them. You have several options:
+
+.. code-block:: bash
+
+    $ nosetests -v pbxplore/test
+
+.. code-block:: bash
+
+    $ python setup.py nosetests
+
+
+Or within the `PBxplore` package:
```

Comment:
Does that mean that you must be in the PBxplore directory to tun this command? If not, the sentence is unclear. I would suggest something like:

    Or from the `PBxplore` module